### PR TITLE
Update Ruby repository guide from Subversion to Git (id)

### PR DESCRIPTION
**Pull request by meisyal**
Changes:

* Update Ruby repository guide as mentioned on #2064 
* Update the TOC 93c64c4
* Fix address of git remote for committers 0b54a1d

### CODE REVIEW

File: id/community/ruby-core/index.md [hunk 1]
```diff
@@ -12,65 +12,40 @@ Sehingga, dari mana Anda memulai?
 
 Topik seputar pengembangan Ruby yang dibahas disini adalah:
 
-* [Menggunakan Subversion untuk Memonitor Pengembangan Ruby](#following-ruby)
-* [Bagaimana Menggunakan Git pada Repository Utama Ruby](#git-ruby)
+* [Menggunakan Git untuk Memonitor Pengembangan Ruby](#following-ruby)
 * [Menyempurnakan Ruby, Patch by Patch](#patching-ruby)
-* [Peraturan untuk Developer Inti](#coding-standards)
+* [Peraturan untuk Core Developer](#coding-standards)
```
Comment (gozali) [inline id/community/ruby-core/index.md]: Why do we change this?

Comment (meisyal) [reply id/community/ruby-core/index.md]: I don't know. I feel "Core Developer" is better than "Developer Inti". What do you think @gozali?

Well, "Developer Inti" also makes sense.

Comment (meisyal) [reply id/community/ruby-core/index.md]: Hi @gozali, any update on this?

Comment (gozali) [reply id/community/ruby-core/index.md]: OK cool

**Pull request merged**